content/departments/product-engineering/engineering/career-development/talent-review-process.md

```diff
@@ -64,13 +64,19 @@ Our [career development framework](framework.md) defines the levels that exist a
 
 ## Promotion requests
 
-Promotion requests are made immediately following the [talent review calibration meeting](#calibration-meeting).
+During the [talent review calibration meeting](#calibration-meeting) managers indicate which teammates they would like to considered for promotion. These teammates are then evaluated against their peers at the level they might get promoted to so that the manager can confirm they are indeed performing at the right level for promotion.
```

Comment:
```suggestion
During the [talent review calibration meeting](#calibration-meeting) managers indicate which teammates they would like considered for promotion. These teammates are then evaluated against their peers at the level they might get promoted to so that the manager can confirm they are indeed performing at the right level for promotion.
```